distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -609,23 +612,17 @@ type Decimal
                     msg = "round cannot accept " + self.to_text
                     Error.throw (Arithmetic_Error.Error msg)
                 False ->  check_round_input self <|
-                    decimal_result = case use_bankers of
-                        False ->
-                            scale = 10 ^ decimal_places
-                            ((self * scale) + 0.5).floor / scale
-                        True ->
-                            ## If the largest integer <= self is odd, use normal
-                               round-towards-positive-infinity rounding; otherwise,
-                               use round-towards-negative-infinity rounding.
-                            scale = 10 ^ decimal_places
-                            scaled_self = self * scale
-                            self_scaled_floor = scaled_self.floor
-                            is_even = (self_scaled_floor % 2) == 0
-                            case is_even of
-                                False ->
-                                    (scaled_self + 0.5).floor / scale
-                                True ->
-                                    (scaled_self - 0.5).ceil / scale
+                    decimal_result =
+                        scale = 10 ^ decimal_places
+                        scaled = self * scale
+                        round_base = scaled.floor
+                        round_down = round_base / scale
+                        round_midpoint = (round_base + 0.5) / scale
+                        round_up = (round_base + 1.0) / scale
```

Comment:
There are some values here that are always calculated even if they're not used; avoiding this would make the code rather longer, but it might be worth it (or worth benchmarking).

Reply:
Done.